app/javascript/components/users/SignInComponent.tsx

```diff
@@ -26,18 +26,20 @@ export interface SignInComponentProps {
 	 * @memberof SignInComponentProps
 	 */
 	onFailure?: (error: SignInError) => void;
+	setLoginState?: (newState: String) => void;
```

Comment:
Why is `setLoginState` added here? It's not in the specification so I want to understand what you'll trying to accomplish.

Reply:
This is how we are adding communication between parent and child component. Should we remove it?

Reply:
What are you trying to communicate between the parent and children?

Reply:
Please remove this.